tests/e2e/postgres_test.go

```diff
@@ -0,0 +1,166 @@
+// +build e2e
```

Comment:
this can be removed as it's not an e2e test. e2e tests by definition do not test dependencies so having an e2e test called `TestPostgres` is a contradiction by itself.

Reply:
Yes, misunderstanding about what is the difference between e2e and acceptance tests.
And maybe this test has no reason to exist at all? Or another category of tests that make sure that our interfaces works with the real backend system before using for mocking.

Reply:
This has been removed entirely and will be added back once we refactor the acceptance/integration tests into 1 test runner.

Reply:
We decided to include in the acceptance (that will be renamed to integration), I'll integrate it when we make the acceptance test run only 1 test environment. (currently, for my test, I created a parallel test env, we need to merge)